src/main/java/com/snowypeaksystems/mobactions/listener/EventListener.java

```diff
@@ -61,14 +62,21 @@ public void onMobDamage(EntityDamageEvent event) {
       return;
     }
 
-    IInteractiveMob mob = ma.getInteractiveMob((LivingEntity) event.getEntity());
-    if (event instanceof EntityDamageByEntityEvent
-        && ((EntityDamageByEntityEvent) event).getDamager() instanceof Player) {
-      MobActionsUser user = ma.getPlayer((Player) ((EntityDamageByEntityEvent) event).getDamager());
-      processEvent(user, mob, event);
-    } else if (mob.exists()) {
-      event.setCancelled(true);
+
+
+    if (event instanceof EntityDamageByEntityEvent) {
+      IInteractiveMob mob = ma.getInteractiveMob((LivingEntity) event.getEntity());
+      Entity damager = ((EntityDamageByEntityEvent) event).getDamager();
+
+      if (damager instanceof Player) {
+        MobActionsUser user = ma.getPlayer((Player) damager);
+        processEvent(user, mob, event);
+      } else if (mob.exists() || ma.getInteractiveMob((LivingEntity) damager).exists()
```

Comment:
This change will cause the event to no longer be cancelled if the event is only an `EntityDamageEvent`. There's also a possibility that the damager is not a LivingEntity, which would cause a `ClassCastException`.